docs/dev/generator.md

```diff
@@ -46,11 +46,14 @@ This is achieved through `adapter.configure_model(content_type: ContentTypeStr)`
 - `identifiers`: A list of fields identifiable as unique references in the source data.
 - `default_reference`: A `RecordData` dictionary of default values to reference this model. This is useful when the source data does not provide a reference that is required in Nautobot.
 - `extend_content_type`: Define this when a source model extends another source model to merge into a single Nautobot model.
+- `forward_references`: Define to forward references to another content type. This is useful when the source data references a content type that is not directly related to the Nautobot content type. For example, a source data references a `dcim.location` content type, but the Nautobot content type is `dcim.locationtype`.
+- `disable_related_reference`: Define, to disable storing references to this content type from other content types. This is useful for e.g. `ObjectChange` model.
+- `pre_import`: Define a callable to be executed before importing the source data. Can be used to alter or cache the source data before importing.
```

Comment:
Would be good to document:
- the args that the callable should accept
- the fact that it actually gets called twice during the import process

Reply:
Fixed